engine/src/main/scala/cromwell/services/EngineMetadataServiceActor.scala

```diff
@@ -28,6 +28,9 @@ object EngineMetadataServiceActor {
 
   val MetadataSummaryTimestampMinimum =
     ConfigFactory.load().getStringOption("services.MetadataService.metadata-summary-timestamp-minimum") map OffsetDateTime.parse
+
+  // Number of runs of the workflow summary actor to allow a workflow ID to sit in the cache before being expired out.
```

Comment:
Combining it w/ the code that uses it it makes more sense but I didn't really find this message to be super helpful. I'm not sure if it's the wording here, the lack of comment below, or some combination of the two ... I realize that's super helpful.